src/main/scala/carldata/series/Sessions.scala

```diff
@@ -26,24 +26,22 @@ object Sessions {
     }.reverse
   }
 
-  /** Finds a session with a duration tolerance
-    * @param maxTolerance the tolerance between two samples in minutes, for example:
-    *                       for a fragment of signal values distributed evenly for exactly one second
-    *                       (2, 0, 1, 0, 0, 2, 3, 4, 5, 7, 0)
-    *                       for Duration.ofSeconds(0) will be following sessions detected: [Session(0,0), Session(2,2), Session(5, 9)]
-    *                       for Duration.ofSeconds(1) will be following sessions detected: [Session(0,2), Session(5, 9)]
+  /** Finds sessions with a duration tolerance
+    * @param tolerance the time distance (inclusive) between two detected sessions that will make the sessions merged,
+    *                  for example, for a fragment of signal values distributed evenly for exactly one second
+    *                  (2, 0, 1, 0, 0, 2, 3, 4, 5, 7, 0)
+    *                  the Duration.ofSeconds(1) will be following sessions detected: [Session(0,0), Session(2,2), Session(5, 9)]
+    *                  the Duration.ofSeconds(2) will be following sessions detected: [Session(0,2), Session(5, 9)]
     * */
 
-  def findSessions[V: Numeric](ts: TimeSeries[V], maxTolerance: Duration)(implicit num: Fractional[V]): Seq[Session] = {
-    if (maxTolerance.compareTo(Duration.between(ts.head.get._1, ts.last.get._1)) >= 0)
+  def findSessions[V: Numeric](ts: TimeSeries[V], tolerance: Duration)(implicit num: Fractional[V]): Seq[Session] = {
+    if (tolerance.compareTo(Duration.between(ts.head.get._1, ts.last.get._1)) >= 0)
```

Comment:
I wonder if this line is needed.
I know that without it some tests won't pass, but should they?